.github/workflows/e2e_tests.yml

```diff
@@ -11,6 +11,84 @@ permissions:
   id-token: write
 
 jobs:
+  azure-e2e-test:
+    name: Run end-to-end tests of Pulumi Kotlin Azure
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      id-token: write
+    steps:
+      - name: Check out project sources
+        uses: actions/checkout@v2
+      - name: Set up Gradle
+        uses: gradle/gradle-build-action@v2
+      - name: Set up Pulumi CLI
+        uses: pulumi/setup-pulumi@v2
+      - name: Authenticate to Azure
+        uses: azure/login@v1
+        id: auth
+        with:
+          # These values were copied from jvm-lab/identity-federation stack's output
+          # The Azure tenant/subscription that is used here belongs to @myhau (Michal Fudala)
+          tenant-id: '1ff3e4bd-41d1-4df2-b776-8d3fcbf77184' # tenant-id output of jvm-lab/identity-federation/dev stack
+          subscription-id: '3565bcd1-dd9b-45a8-9d7c-8880aaaa9a9f' # subscription-id output of jvm-lab/identity-federation/dev stack
+          client-id: '4169b156-839a-48c2-846f-d7c079331c71' # service-principal-client-id output of jvm-lab/identity-federation/dev stack
+      - name: Publish Pulumi Kotlin Azure to Maven Local
+        run: ./gradlew publishPulumiAzureE2ePublicationToMavenLocal -x formatKotlinPulumiAzure -x dokkaPulumiAzureJavadoc
```

Comment:
Remove ` -x formatKotlinPulumiAzure -x dokkaPulumiAzureJavadoc` (they don't get triggered with the E2E task)